### DISCUSSION TITLE
How do I install this? I have NO idea what i'm doing...

### DISCUSSION THREAD

**Discussion by VoxNiphili** [acheong08/ChatGPT#441] [Q&A]
Is it like... command prompt? Do I need to install Python??

**Comment by acheong08** [comment]
This repository is not meant for non-developers. You would save yourself some sanity if you don't touch this spaghetti code.

**Reply by acheong08** [reply]
But yes, you need Python installed

**Reply by acheong08** [reply]
Ask ChatGPT how to install Python and pip
